### ISSUE TITLE
hpdf_version.h is not included by hpdf.h

### ISSUE THREAD

**Issue by opoplawski** [libharu/libharu#246]
plplot's libharu version check depends on the HPDF_*_VERSION macros being set after including hpdf.h.  This appears to no longer be the case as hpdf_version.h is not included by any other header.  Is this really intentional?  It seems an unnecessary change.

**Comment by opoplawski** [comment]
This also affects saga which expects hpdf.h to define HPDF_VERSION_TEXT:
```
/bin/sh ../../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../../..    -fPIC -I../../../../src/saga_core -D_SAGA_LINUX -D_TYPEDEF_BYTE -D_TYPEDEF_WORD   -fopenmp -lgomp -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64  -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o doc_pdf.lo doc_pdf.cpp
libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../../.. -fPIC -I../../../../src/saga_core -D_SAGA_LINUX -D_TYPEDEF_BYTE -D_TYPEDEF_WORD -fopenmp -lgomp -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c doc_pdf.cpp  -fPIC -DPIC -o .libs/doc_pdf.o
doc_pdf.cpp: In static member function 'static const wchar_t* CSG_Doc_PDF::Get_Version()':
doc_pdf.cpp:147:105: error: 'HPDF_VERSION_TEXT' was not declared in this scope
```

**Comment by bramton** [comment]
This has been fixed in the `devel` branch by PR #241 
I'll make a minor release next week as I guess this will be an recurring issue.
